src/main/scala/cromwell/Main.scala

```diff
@@ -53,29 +39,33 @@ object Main extends App {
     )
 
     binding.workflow.calls.foreach { call =>
-      println(s"Call: $call")
```

Comment:
Cool, the more of this unused and untested code gets deleted, the better our coverage numbers!  :smile: